Sources/Operators.swift

```diff
@@ -204,6 +245,18 @@ public prefix func -(offset: UIOffset) -> UIOffset {
     return UIOffset(horizontal: -offset.horizontal, vertical: -offset.vertical)
 }
 
+public func +(lhs: CGSize, rhs: UIOffset) -> CGSize {
+    return CGSize(width: lhs.width + rhs.horizontal, height: lhs.height + rhs.vertical)
+}
+
+public func +(lhs: UIOffset, rhs: CGSize) -> CGSize {
+    return CGSize(width: rhs.width + lhs.horizontal, height: rhs.height + lhs.vertical)
+}
+
+public func -(lhs: CGSize, rhs: UIOffset) -> CGSize {
+    return CGSize(width: lhs.width - rhs.horizontal, height: lhs.height - rhs.vertical)
+}
+
```

Comment:
I like it.  Wouldn't be surprised if people complain, but I think it adds a bit of clarity that'll help in debugging.